specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -106,41 +106,1303 @@
+        "operationId": "TimeSeries_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/timeSeriesId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TimeSeriesCreateRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a new time series example": {
+            "$ref": "./examples/TimeSeriesCreate.json"
+          }
+        }
+      },
+      "delete": {
+        "summary": "Delete the specified timeseries.",
+        "description": "Delete the specified timeseries.",
+        "operationId": "TimeSeries_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/timeSeriesId"
+          }
+        ],
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a time series example": {
+            "$ref": "./examples/TimeSeriesDelete.json"
+          }
+        }
+      }
+    },
+    "/timeseries": {
+      "get": {
+        "summary": "List time series of each unique user.",
+        "description": "List time series of each unique user.",
+        "operationId": "TimeSeries_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/next"
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/TimeSeriesList"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get information of a time series example": {
+            "$ref": "./examples/TimeSeriesList.json"
+          }
+        }
+      }
+    },
+    "/timeseries/{timeSeriesId}/write": {
+      "put": {
```

Comment:
Consider providing batch write API to write multiple time series points together

Reply:
Resolved. We won't provide batch write as we are using Azure Table. But Azure table don't support batch write so we try to avoid batch write scenario